src/dialog/dlgabout.cpp

```diff
@@ -40,67 +40,47 @@ DlgAbout::DlgAbout(QWidget* parent) : QDialog(parent), Ui::DlgAboutDlg() {
 
     QStringList thisReleaseDevelopers;
     thisReleaseDevelopers
+            << "Daniel Sch&uuml;rmann"
```

Comment:
This list was previously sorted by tenure on the dev team (i.e. we just append new people to it and remove inactive folks). What's this ordering?

Reply:
By number of commits for the current release

Reply:
I can change it back if there are objections. I don't really care either way. I just thought it made sense to order this the same way as the contributor list below.

Reply:
Yea, I thought so, but it didn't match anything I could come up with using your `git shortlog` command from below.

I don't think commits are the right metric. It's easy to generate a lot of commits (just make a lot of mistakes :P). Commits also doesn't reward other types of work, like maintaining CI or build infrastructure, doing code review, writing design docs, or community outreach via the forums.

I think this is the sort of change ought to be discussed on Zulip first.

Reply:
Maybe we can sort by lines of code removed :P

Reply:
Can you just revert contributor names related commits?

It was IMHO already in a good shape. Thanks names are just ordered by the added date without any additional meaning. 
If we consider a top place better than a place below, we open the door for conflicts. 

We have also decided to keep a name on the contributor list for at least two years. Only if one was absence longer it is fair to move him to past developers.

Reply:
The code says "thisReleaseContributors", although I see now the string shown in the GUI is "With contributions from". I don't know where you got 2 years from, but that seems random to me. It makes sense to me to credit contributors to the current release.

> Thanks names are just ordered by the added date without any additional meaning.

I do like the idea to not sort by some metric that imperfectly correlates with "amount of work contributed", but date of first contribution would be difficult to track. We could simply sort alphabetically, which is easy to do with `git shortlog -s` without the `-n` option.

Reply:
> Yes, commits are a crude metric, but isn't any metric? I suppose lines of code changed might be a slightly better metric, but often times less code is better code...

Commits are too close to a score that people can get driven towards optimizing, and if at the end of every release there's a "leaderboard" as names are re-ranked, that just makes it seem like too much of a competition and could imply value comparisons between people's contributions.

I also think that contribution can be more than writing code, and counting commits is a statement that code is the primary value people bring to the team. And the ranking doesn't include contributions to the manual, website, or build server repos (I don't think you included those?) so that's also a statement about what type of contribution is valued. There's also work that doesn't produce commits (user support, launchpad bug wrangling, CI and build server maintenance, release planning / cat herding, etc.). 

> Maybe we can sort by lines of code removed :P

Now that I can get behind!

Reply:
> but date of first contribution would be difficult to track

I'm pretty sure git commits include a date ;). Though it can be falsified.

Though what it doesn't contain is the user's desired name in the credits. When we get a PR we need to remember to ask them for that.

Reply:
I like projects that list contributors alphabetically, +1 to that.

Reply:
I don't think we have a problem here. In the past I have carefully taken care that every new contributor is listed in the about box after their first commit. Every new contributor was added to the bottom. Before a release, we have decided who was moved to the past section. This means that naturally remaining contributors are moved up. 
The list was still valid. We have no need to reorder it away from this natural order. We also need no algorithm for it it just happens. We just need to fix the commit by accident.

Reply:
Preparing a PR

Reply:
https://github.com/mixxxdj/mixxx/pull/1949